pkgs/development/python-modules/torchdata/binary-hashes.nix

```diff
@@ -0,0 +1,66 @@
+# Warning: Need to update at the same time as pytorch-bin
+#
+# Precompiled wheels can be found at:
+# https://pypi.org/project/torchdata/#files
```

Comment:
Normally I would try and get this from https://download.pytorch.org/ but I couldn't find it there